test/e2e/helpers.js

```diff
@@ -315,6 +315,7 @@ const importSRPOnboardingFlow = async (driver, seedPhrase, password) => {
   await driver.fill('[data-testid="create-password-confirm"]', password);
   await driver.clickElement('[data-testid="create-password-terms"]');
   await driver.clickElement('[data-testid="create-password-import"]');
+  await driver.waitForElementNotPresent('.loading-overlay');
```

Comment:
Without this, if the first vault encryption takes too long on the device, there's a chance that the onboarding completion screen will be visible, but the loader will keep spinning for some more time:

![Screenshot 2023-11-23 at 15 38 30](https://github.com/MetaMask/metamask-extension/assets/34438276/bea8c0fc-0875-4302-9d8e-3fff4588e081)

 

While this is unclear to me, tests should mind this, or they will fail with this error:

`Element <button class="hw-connect__btn"> is not clickable at point (683,341) because another element <div class="mm-box loading-overlay"> obscures it`

Reply:
Perhaps we could extract this to a function whose name reflects this need in the future if we forget this, but this makes sense for now!